### DISCUSSION TITLE
Should we put the bug-report link directly at the top of the page?

### DISCUSSION THREAD

**Discussion by rjsparks** [ietf-tools/datatracker#3888] [Ideas]
Chairs continue to ask where to report bugs, even with the links both in the Other menu and at the bottom of the page.

Would this help:
![image](https://user-images.githubusercontent.com/10996692/165358918-f2b6ecaf-6ab4-4466-8f55-97cd007e45f5.png)
or is it a step too far?

**Comment by EnochianD** [comment]
It should be easily accessible to get to.

**Comment by larseggert** [comment]
Can do. We might need to shrink the user menu (by renaming it) to make room for it horizontally.
